shared-libs/rules-engine/test/pouchdb-provider.spec.js

```diff
@@ -68,9 +68,10 @@ describe('pouchdb provider', () => {
 
   describe('allTasks', () => {
     it('for owner', async () => expect(await pouchdbProvider(db).allTasks('owner')).excludingEvery('_rev')
-      .to.deep.eq([headlessTask, taskOwnedByChtContact]));
+      .to.have.deep.members([headlessTask, taskOwnedByChtContact, taskRequestedByChtContact]));
```

Comment:
Is it possible to use `to.deep.eq` still? I think this assertion would pass even if the fn returned docs it shouldn't.

Reply:
`to.have.deep.members` passes if the two arrays have the same members in deep equality, regardless of order.
`to.include.deep.members` would have the behavior you're describing.
`to.deep.equal` passes if the two arrays have the same members in the same order. 

I hate it when you have to change assertions when results come in in different order when order is not relevant to the result.